airflow/dags/quarterly_tasks/go_live.py

```diff
@@ -0,0 +1,59 @@
+import configparser
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+from airflow.sensors.external_task import ExternalTaskSensor
+from slack_sdk import WebClient
+from updater.callbacks import airflow_task_failure, airflow_task_success
+from reporting_database_generator.database import validate_query
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'email': ['bcard@air.org'],
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 0,
+    'retry_delay': timedelta(minutes=5),
+    'concurrency': 4,
+    'queue': 'data_collector'
+}
+class SQLTemplatedPythonOperator(PythonOperator):
+    template_ext = ('.sql',)
+
+go_live_dag = DAG("GO_LIVE"
+                       , default_args=default_args
+                       , start_date=datetime(2023, 10, 1)
```

Comment:
If you set the start date to Oct 1 2023 with quarterly schedule, the first run will be on Jan 1 2024.

Reply:
Again just for testing because I didn't want these tasks to run yet. This is a placeholder DAG for now until someone can devote time to GO LIVE automation